app/controllers/parties_controller.rb

```diff
@@ -1,6 +1,51 @@
 class PartiesController < ApplicationController
+  before_action :require_user
+
   def new
+    response = conn.get("3/movie/#{params[:movie_id]}?api_key=#{movies_api_key}")
+    @movie = JSON.parse(response.body, symbolize_names: true)
+    @party = Party.new
+  end
+
+  def create
+    movie = set_movie(party_params[:movie_id], params[:party][:movie_title])
+    party = Party.create(party_params)
+    if party.save
+      valid_party(party)
+    else
+      invalid_party(party, movie.id)
+    end
+  end
+
+  private
+
+  def party_params
+    params.require(:party).permit(:duration, :party_date, :start_time, :movie_id)
+  end
+
+  def conn
+    Faraday.new(url: 'https://api.themoviedb.org')
   end
 
-  def create; end
+  def movies_api_key
+    ENV['MOVIES_API_KEY']
+  end
+
+  def valid_party(party)
+    party.party_users.create(user_id: current_user.id, attendee_type: 0)
```

Comment:
Creates party user record for the host. Once we add Friends, we'll have to update this to also add party user records for any invited friends.